local.properties

```diff
@@ -0,0 +1,9 @@
+## This file must *NOT* be checked into Version Control Systems,
```

Comment:
This warning should not have been ignored. The build fails for every machine that doesn't have their Android sdk/ndk in the specified paths.

Reply:
Thanks, fixed